### DISCUSSION TITLE
Task list: Possibility to add a small description from the conversion

### DISCUSSION THREAD

**Discussion by an0rak-dev** [community/community#4429] [Projects and Issues] [answered]
Hi all ! 

It might help to add a way (like a popup ?) to add a description to the issue when converting a task as an issue.

For example in my use case, I did a work planning using task list, on purpose I used short sentences, then after all my planning I tried to change some of them as Issues, and face the fact that I want to add more details in them. For the moment the only way to do this is to : convert the task, then navigate to the issue, Edit the description, save the change, and then go back to the original issue.

Maybe it could be interesting to reduce the amount of user interaction to do this in case a task need details ?

**Comment by labrose** [comment, marked as answer]
There are [two keyboard shortcuts](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#issues-and-pull-requests) that will take you to a full issue authoring experience rather than converting an issue inline 😄 

<kbd>shift</kbd> click will open a new tab with the issue create experience while <kbd>alt</kbd> click will open the issue create experience in your current tab:

![2021-07-01 07 45 49](https://user-images.githubusercontent.com/22751162/124126708-8e97b080-da40-11eb-8f05-0f3876501019.gif)

We could definitely do a better job increasing the visibility of this functionality. I would also be interested to know if you use issue templates and how that may or may not be helpful to integrate for you?

**Reply by bennycode** [reply]
@labrose thanks for showing us these options. 

I started using task lists with an issue that my team and I created 6 months ago: https://github.com/wireapp/wire-webapp/issues/10279

We have a huge list with 180 tasks. It's great that we can create real issues now out of these tasks to keep track of their individual progress. I clicked the "Convert to issue" button and was amazed that it immediately created (and linked) a GitHub issue out of a list item. But just like @an0rak-dev I was missing a pop-up to add a more descriptive issue title. 

The "Alt" + "Click" functionality solves this usability problem. Can you add this hint to the "Convert to issue" tooltip so others find it quickly as well?

**Reply by an0rak-dev** [reply]
Hi @labrose ! 

Thanks for the tip, didn't know about this feature ! I'll give it a try today 😄 

I don't use Issue Template in my repo for the moment (it's a WIP private one), but when the repo will be open, I'll add issue templates. 

Thanks any way 👏
